components/error-summary/src/fixtures.js

```diff
@@ -18,11 +20,16 @@ export const errors = [
   },
 ];
 
+const onHandleErrorClick = (targetName) => {
```

Comment:
Don't know why I didn't think of this, way better, nice one! I think we could iterate on this in the future if we wanted to also... althoooooo context api : D